app/controllers/concerns/blacklight/bookmarks.rb

```diff
@@ -136,6 +134,10 @@ def clear
 
   protected
 
+  def search_service
+    Blacklight::SearchService.new(blacklight_config)
```

Comment:
Would it be useful to make the search service class settable?  Just thinking about if an app or a gem might want to provide their own search service class to be used.

Totally understand if that seems premature though.  Fine with letting the use case reveal itself later.